instance_test.go

```diff
@@ -104,18 +105,54 @@ var _ = Describe("Instance", func() {
 				Expect(instance.Name).To(Equal("INSTTEST"), "name")
 				Expect(instance.Directory).To(Equal("/ensemble/instances/insttest/"), "directory")
 				Expect(instance.Version).To(Equal("2015.2.2.805.0.16216"), "version")
-				Expect(instance.Status).To(Equal(isclib.InstanceStatusRunning), "status")
+				Expect(instance.Status).To(Equal(InstanceStatusRunning), "status")
 				Expect(instance.Activity).To(Equal("since Fri May 13 22:07:02 2016"), "activity")
 				Expect(instance.CPFFileName).To(Equal("cache.cpf"), "cpf")
 				Expect(instance.SuperServerPort).To(Equal(56772), "ss port")
 				Expect(instance.WebServerPort).To(Equal(57772), "ws port")
 				Expect(instance.JDBCPort).To(Equal(62972), "jdbc port")
 				Expect(instance.State).To(Equal("ok"), "state")
-				Expect(instance.Product).To(Equal(isclib.Cache), "product")
+				Expect(instance.Product).To(Equal(Cache), "product")
 				Expect(instance.MirrorMemberType).To(Equal("Failover"), "mirror type")
 				Expect(instance.MirrorStatus).To(Equal("Primary"), "mirror status")
 				Expect(instance.DataDirectory).To(Equal("/mgr/config"), "data directory")
 			})
 		})
 	})
+	Context("WaitForReady", func() {
+		var i int
+		BeforeEach(func() {
+			i = 0
+		})
+		Context("With timeout", func() {
+			Context("Does not come up", func() {
+				instance, err := InstanceFromQList("INSTTEST^/ensemble/instances/insttest/^2012.2.3.903.2.12515^down, last used Thu Sep 15 18:58:30 2016^cache.cpf^56772^57772^62972")
+				ctx, can := context.WithTimeout(context.Background(), 50*time.Millisecond)
+				defer can()
+				err = instance.WaitForReady(ctx)
```

Comment:
I recognize that these tests are most likely just following the pattern that I initially coded.  Unfortunately, that pattern goes against proper practices when coding Ginkgo tests.

From the documentation...

> In general, the only code within a container block should be an It block or a BeforeEach/JustBeforeEach/AfterEach block, or closure variable declarations.

> It is also a mistake to initialize a closure variable in a container block. If one of your Its mutates that variable, subsequent Its will receive the mutated value. This is a case of test pollution and can be hard to track down. Always initialize your variables in BeforeEach blocks.

Since existing tests are already coded this way, I logged case #5 to make a pass through the tests and restructure them all.